src/account/AccountInfoQuery.js

```diff
@@ -156,18 +178,59 @@ export default class AccountInfoQuery extends Query {
      */
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     _mapResponse(response, nodeAccountId, request) {
-        const info =
-            /** @type {HashgraphProto.proto.ICryptoGetInfoResponse} */ (
-                response.cryptoGetInfo
+        return new Promise((resolve, reject) => {
+            const mirrorNodeGateway = MirrorNodeGateway.forNetwork(
+                this._mirrorNetwork,
+                this._ledgerId,
             );
 
-        return Promise.resolve(
-            AccountInfo._fromProtobuf(
-                /** @type {HashgraphProto.proto.CryptoGetInfoResponse.IAccountInfo} */ (
-                    info.accountInfo
-                ),
-            ),
-        );
+            const mirrorNodeService = new MirrorNodeService(mirrorNodeGateway);
+
+            const info =
+                /** @type {HashgraphProto.proto.ICryptoGetInfoResponse} */ (
+                    response.cryptoGetInfo
+                );
+
+            if (info.accountInfo && info.accountInfo.accountID) {
```

Comment:
Otherwise it would be highlighted:
```
'accountInfoFromConsensusNode.accountInfo' is possibly 'null' or 'undefined'.ts(18049)
Argument of type 'IAccountID | null | undefined' is not assignable to parameter of type 'IAccountID'.
  Type 'undefined' is not assignable to type 'IAccountID'.ts(2345)
```